packages/template-retail-react-app/app/pages/checkout/partials/shipping-multi-address.jsx

```diff
@@ -446,42 +447,65 @@ const ShippingMultiAddress = ({
+                                                        _focus={{
+                                                            borderColor: 'blue.500',
+                                                            boxShadow:
+                                                                '0 0 0 1px var(--chakra-colors-blue-500)'
+                                                        }}
+                                                    >
+                                                        {addresses.length === 0 ? (
+                                                            <option value="">
+                                                                {formatMessage({
+                                                                    id: 'shipping_multi_address.no_addresses_available',
                                                                     defaultMessage:
-                                                                        '{city}, {stateCode} {postalCode}'
-                                                                },
-                                                                {
-                                                                    city: addr.city,
-                                                                    stateCode: addr.stateCode || '',
-                                                                    postalCode: addr.postalCode
-                                                                }
-                                                            )}
-                                                        </option>
-                                                    ))}
-                                                    <option value={ADD_NEW_ADDRESS_OPTION_VALUE}>
-                                                        + {formatMessage(addNewAddressLabel)}
-                                                    </option>
-                                                </Select>
+                                                                        'No Address Available'
+                                                                })}
+                                                            </option>
+                                                        ) : (
+                                                            addresses.map((addr) => (
+                                                                <option
+                                                                    key={addr.addressId}
+                                                                    value={addr.addressId}
+                                                                >
+                                                                    {addr.firstName} {addr.lastName}{' '}
+                                                                    - {addr.address1},{' '}
+                                                                    {formatMessage(
```

Comment:
Hmm why do we need `formatMessage` here? the data is returned from the api and there is nothing we need to translate so to speak. If I understand correctly you want to format the string to a certain pattern which we can do normal with string literal. I'd suggest we do that for entire address string using string [literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)